packages/buffetjs-core/src/components/DatePicker/index.js

```diff
@@ -119,17 +144,20 @@ class DatePicker extends React.PureComponent {
             tabIndex={tabIndex}
           />
         </div>
-        {visible && (
-          <DayPickerSingleDateController
-            date={date}
-            focused={isFocused}
-            numberOfMonths={1}
-            keepOpenOnDateSelect
-            onFocusChange={this.handleFocusChange}
-            onDateChange={this.handleDateChange}
-            onOutsideClick={this.handleOutsideClick}
-          />
-        )}
+        <div>
+          {isFocused && (
```

Comment:
Why not keeping `visible`?

Reply:
I added the `initialVisibleMonth` prop to specify the month and year panel to display on focus. It only works if the datepicker prop `focused` is true.

`visible` manages the input format (MM/DD/YYYY or more human-readable format) and was also used to display the `DayPickerSingleDateController`.

In our case, we want enable the user to edit manually the date from the text input. But to let the panel to be updated when the user changes the input, we don't want the format to change (in case the user haven't finish to edit). 

As the `initialVisibleMonth` value changes but doesn't rerender the `DayPickerSingleDateController`, I have to close and reopen the panel (only after the user finish to change the text input). 

It's why I used two attributes.